src/ServiceControl.Audit/Auditing/AuditIngestion.cs

```diff
@@ -76,18 +76,18 @@ async Task EnsureStarted(CancellationToken cancellationToken)
                 await startStopSemaphore.WaitAsync(cancellationToken);
                 logger.Debug("Ensure started. Start/stop semaphore acquired");
 
-                var canIngest = unitOfWorkFactory.CanIngestMore();
-
-                logger.DebugFormat("Ensure started {0}", canIngest);
-
-                if (canIngest)
-                {
-                    await SetUpAndStartInfrastructure(cancellationToken);
-                }
-                else
+                if (!unitOfWorkFactory.CanIngestMore())
                 {
+                    logger.Warn("Ensure started. Storage unable to ingest more. Ingestion will be paused");
```

Comment:
@andreasohlund already is done in these methods, now we're double logging

Reply:
we are only logging INFO in there, I think we should log INFO if we stop due to a requested shutdown and WARN when we stop due to the persister being unhealthy?

Reply:
This code doesn't know if the persister is unhealthy. You mean when `unitOfWorkFactory.CanIngestMore()` returns `false`?

I'm ok with such a warning but as a state change. This method is invoked I think every minute so those logs would be written every cycle.

Reply:
Yea we can tailor the message to say something like "Persister unable to ingest more data, pausing" or something.

Thinking about this more, why are we not using the circuit breaker that we use elsewhere here?